### PR TITLE
Translation to Bengali

### PR DESCRIPTION
**Pull request by eNipu**
- This pull request contains the translation of Chapter 0 in Bengali
- The translation of `_toctree.yml` file (chapter 0-1)
The other chapters are WIP.

### CODE REVIEW

File: chapters/bn/_toctree.yml [hunk 1]
```diff
@@ -0,0 +1,173 @@
+- title: 0. সেটআপ
+  sections:
+  - local: chapter0/1
+    title: ভূমিকা
+
+- title: 1. ট্রান্সফরমার মডেল
```
Comment (lewtun) [inline chapters/bn/_toctree.yml:6]: Thanks for translating the whole Table of Contents, but we should only include the sections that have a corresponding set of translated MDX files. In this case, we should remove everything from this line downwards

**Review by lewtun — APPROVED** [review]
Thank you for this contribution @eNipu ! We just need to tweak the `_toctree.yml` file, but otherwise this looks good to go!

File: chapters/bn/_toctree.yml (hunk 1 above)
Comment (eNipu) [reply chapters/bn/_toctree.yml:6]: @lewtun Thanks for the quick review. Got it 👍🏾 
I just changed accordingly. Hope this will fix the issue.

Comment (lewtun) [reply chapters/bn/_toctree.yml:6]: Oh sorry, what I meant is that `_toctree.yml` should only include this:

```
- title: 0. সেটআপ
  sections:
  - local: chapter0/1
    title: ভূমিকা
```

Everything else should be removed until we have the corresponding MDX files translated. Would you mind fixing that? Sorry for the bother!

File: chapters/bn/chapter0/1.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+## Google Colab নোটবুক ব্যবহার করার পদ্ধতি
+
+Colab নোটবুক ব্যবহার করার সবচেয়ে সহজ সেটআপ হচ্ছে ব্রাউজারে একটি নোটবুক ওপেন করুন এবং সরাসরি কোডিং এ যান!
+
+আপনি যদি Colab-এর সাথে পরিচিত না হন তাহলে আমরা আপনাকে [Colab পরিচয়](https://colab.research.google.com/notebooks/intro.ipynb) অনুসরণ করে শুরু করার পরামর্শ দিচ্ছি। Colab আপনাকে কিছু এক্সেলারেসন হার্ডওয়্যার ব্যবহার করতে দেয়, যেমন GPUs বা TPUs যা ছোট ওয়ার্ক লোডের জন্য ফ্রি।
+
+Colab-এর উপর আপানার হাত চলে আসলে একটি নতুন নোটবুক ওপেন করে সেট-আপ শুরু করুন:
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter0/new_colab.png" alt="An empty colab notebook" width="80%"/ >
```
[the remark this answers is not in the document]
Comment (eNipu) [reply chapters/bn/chapter0/1.mdx]: @lewtun pushed the suggested fix. Will carefully check for future changes.

File: chapters/bn/_toctree.yml (hunk 1 above)
Comment (lewtun) [reply chapters/bn/_toctree.yml]: Thank you for iterating - this looks great!

**Pull request merged**